plugins/Monolog/config/config.php

```diff
@@ -22,7 +22,7 @@
         'screen'   => 'Piwik\Plugins\Monolog\Handler\WebNotificationHandler',
         'database' => 'Piwik\Plugins\Monolog\Handler\DatabaseHandler',
         'errorlog' => '\Monolog\Handler\ErrorLogHandler',
```

Comment:
@AltamashShaikh should we also wrap the error handler? I would assume it could have exactly the same issues for prefixing?

Reply:
Yes we should wrap the error handler, updated code.

Reply:
Please check the PHPCS failure in the new class.

Reply:
Fixed